aws/main.tf

```diff
@@ -71,7 +71,8 @@ module "eks" {
   cluster_endpoint_private_access = true
   vpc_id       = module.vpc.vpc_id
   enable_irsa  = true
-
+  # TODO: pull this out into a variable
+  permissions_boundary = "arn:aws:iam::328656936502:policy/Terraform-Perm-Boundary"
```

Comment:
Should this be a variable?  cluster_roles_permission_boundary?

Reply:
yeah, I've got a bunch of things to pull out into variables.